src/backend/database/idma_axi.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
```

Comment:
This database format needs to be documented. Both in-file and in a dedicated md file.